gammapy/estimators/flux_estimator.py

```diff
@@ -0,0 +1,161 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+import numpy as np
+from astropy import units as u
+from gammapy.datasets import Datasets
+from gammapy.modeling.models import ScaleSpectralModel
+from gammapy.estimators import ParameterEstimator
+
+__all__ = ["FluxEstimator"]
```

Comment:
For now I would suggest to make the `FluxEstimator` not public and keep it as an implementation detail of the `LightCurveEstimator` and `FluxPointsEstimator`, which expose the end-user API.

Reply:
Removed `FluxEstimator` and `ParameterEstimator` from exposed API